pcsx2/IopBios.cpp

```diff
@@ -1007,19 +1026,25 @@ namespace R3000A
 							case 'O':
 							case 'x':
 							case 'X':
```

Comment:
```suggestion
							case 'X':
							case 'u':
```

Mind adding 'u' while you're here? It'll fix some FFX prints.

Reply:
don't forget U as well :)

Might also be worth adding d/D to be sure too

Reply:
added it